src/app/security-group/sg-rules/sg-rules.component.html

```diff
@@ -45,12 +45,13 @@
                 name="cidr"
                 type="text"
                 [(ngModel)]="cidr"
```

Comment:
We do not need two-way binding if we use ngModel and ngModelChange in the same time